workspaces/util/src/__tests__/squashWhitespace-test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { squashWhitespace } from "../squashWhitespace";
+
+describe("squashWhitespace", () => {
+  it("removes whitespaces between words if it exists", () => {
+    expect(squashWhitespace("hello   world")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the end of the string", () => {
+    expect(squashWhitespace("hello world        ")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the beginginng of the string", () => {
+    expect(squashWhitespace("         hello world")).toBe("hello world");
+  });
+
+  it("nothing should be changed", () => {
+    expect(squashWhitespace("hello world !")).toBe("hello world !");
+  });
+
+  it("empty string", () => {
```

Comment:
nit: all the other test titles start with a verb, let's update this one to do the same for consistency